torch/testing/_internal/common_device_type.py

```diff
@@ -511,8 +511,6 @@ def get_device_type_test_bases():
         test_bases.append(CPUTestBase)
         if torch.cuda.is_available():
             test_bases.append(CUDATestBase)
-        elif torch.backends.mps.is_available():
```

Comment:
I would leave a comment here.
Because we have a lot of skips in the other test files just to skip the tests added by this.
Something like:

```python
        # Disable MPS testing in generic device testing temporarily while we're
        # ramping up support.
        # elif torch.backends.mps.is_available():
        #    test_bases.append(MPSTestBase)
```

Reply:
Good catch, done.